demo/src/app/components/accordion/overview/accordion-overview.component.html

```diff
@@ -0,0 +1,114 @@
+    <ngbd-code [snippet]="snippets.basicNoOpts"></ngbd-code>
+  </p>
+
+  <p>
+    The above example uses the <code>title</code> attribute for the ngb-panel element to define the text displayed on the panel
+    header. While this can be customized, this setup provides the basic representation of the title as clickable text.
+  </p>
+
+  <p>
+    The above example also uses the <code>ngbPanelContent</code> directive to denote the container of the content to be displayed in the panel.
+  </p>
+
+  <p>
+    Some additional notes around basic usage of the component are the following:
+  </p>
+
+  <ul>
+    <li>
+      The accordion element can be given a template reference variable, which allows external controls to drive the
+      operation of the accordion (i.e., which panels are open).
+    </li>
+    <li>
+      Each <code>ngb-panel</code> can be given an <code>id</code> attribute, which would allow for direct access to
+      identify or toggle the open status of the given panel.
+    </li>
+    <li>
+      It is also possible to assign the <code>activeIds</code> attribute of the <code>ngb-accordion</code> element to the
+      value of one of the specified <code>ngb-panel</code> element's <code>id</code> to define which panel is open by default.
+    </li>
+    <li>
+      A panel can also be disabled, which prevents it from being able to be opened by using the <code>disabled</code> attribute.
+    </li>
+
+    <li>
+      The accordion itself can also be set to only allow one open panel at a time by using the <code>closeOthers</code> attribute.
+    </li>
+  </ul>
+
+
+</ngbd-overview-section>
+
+<ngbd-overview-section [section]="sections['customization']">
+  <p>
+    Customization Info
+  </p>
+  <p>
+    There are really 2 ways to customize the usage of the accordion.
+  </p>
+  <ul>
+    <li>Custom Panel Headers</li>
+    <li>Custom Global Configuration</li>
+  </ul>
+
+  <h3>Custom Panel Headers</h3>
+
+  <p>
+    <code>ngb-panel</code> supports the <code>title</code> attribute for defining a simple, click-able header with text
+    for the panel which works in many cases.
+  </p>
+
+  <p>
+    However, sometimes it is useful to be able to do something more advanced with the header, such as display custom
+    HTML, or use a button or a specific click-able element to trigger the toggle behavior for the panel.
+  </p>
+
+  <p>
+    To accomplish this, the following directives will be helpful:
+  </p>
+  <ol>
+    <li><code>ngbPanelHeader</code></li>
+    <li><code>ngbPanelToggle</code></li>
+  </ol>
+
+  <p>
+    By using these directives to associate the toggle element and the header content, custom panel headers can easily be implemented.
+    The common way of doing this basically involves defining multiple <code>ng-template</code> blocks, and applying the aforementioned
+    directives.  Let's take a look:
+  </p>
+
+  <h5>Using Bootstrap standard utility classes for a basic, stretched button</h5>
```

Comment:
I would add some spacing before all your `<h5>` either using a `<br />` just before or sticking a `class="mt-5"` on each one of them.